frontend/src/hooks/contracts/useSetHooks.js

```diff
@@ -0,0 +1,8 @@
+
+const useSetHooks = ({ obj }) => {
```

Comment:
Hoping that this isn't needed in this version as setHook will ideally be called by multicall at the time of calling `setBadge` (or the equivalent)